messages/nl/app.php

```diff
@@ -0,0 +1,200 @@
+<?php
+return [
+    /* Hi from Thana */
+    'Ascending' => 'Oplopend',
+    'Add New Server' => 'Voeg Nieuwe Server Toe',
+    'Add Your Server' => 'Voeg je Server Toe',
+    'Add Bot' => 'Voeg De Bot Toe',
+    'Add a Tag' => 'Voeg Een Tag Toe',
+    'Add <b>CREATE_INSTANT_INVITE</b> and <b>SEND_MESSAGES</b> permissions to DISBOARD bot.' => 'Voeg <b>Creëer Directe Uitnodiging</b> en <b>Verstuur Berichten</b> permissies aan de DISBOARD bot.',
+    'Admin response' => 'Admin reactie',
```

Comment:
"I'd remove the space in Admin reacties and servers, we string together words like that"

Reply:
What Austin said. For this one, 'Admin-reacties' or 'Adminreacties' both work.

Reply:
Not to my knowledge. Looking it up shows results of it not being "stringed together" either